src/llm-coding-tools-core/README.md

```
@@ -1,64 +1,254 @@
+[`system_prompt(self, prompt)`]: crate::SystemPromptBuilder::system_prompt
+[`build(self)`]: crate::SystemPromptBuilder::build
+[`context`]: crate::context
+[`ToolContext`]: crate::context::ToolContext
+[`PathResolver`]: crate::PathResolver
+[`AbsolutePathResolver`]: crate::AbsolutePathResolver
+[`AllowedPathResolver`]: crate::AllowedPathResolver
+[`permissions`]: crate::permissions
+[`Rule`]: crate::permissions::Rule
+[`Ruleset`]: crate::permissions::Ruleset
+[`PermissionAction::Deny`]: crate::permissions::PermissionAction::Deny
+[`ToolOutput`]: crate::ToolOutput
```

markdownlint-cli2 (0.20.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "`tooloutput`"

(MD053, link-image-reference-definitions)